notebooks/cuml_benchmarks.ipynb

```diff
@@ -6,59 +6,32 @@
    "source": [
     "# Benchmark and Bounds Tests\n",
     "\n",
-    "The purpose of this notebook is to benchmark all of the single GPU cuML algorithms against their skLearn counterparts, while also providing the ability to find and verify upper bounds.\n",
+    "The purpose of this notebook is to benchmark all of the single GPU cuML algorithms against their skLearn counterparts. This allows validation of correctness as well as the searching of upper bounds.\n",
     "\n",
     "Each benchmark returns a Panda with the results, which can then be analyzed, manipulated, and stored to disk. "
```

Comment:
I would love to refer to a Pandas DF as a "panda" in the future as shorthand, but probably Pandas DataFrame would be clearer here.